### PR TITLE
Doc for PA-2401,2400,2406

### PR DESCRIPTION
**Pull request by devinleighsmith**
Note that a lot of this is dependent on @goemen PR, status, thus the DRAFT PR. Will need to make updates to his CICD pipeline to re enable sonarqube.

### CODE REVIEW

**Review by agalbrai — COMMENTED** [review]
I don't know yaml files very well, but the ReadMes make sense 👍

File: openshift/4.0/templates/sonarqube/sonarqube-postgresql-template.yaml [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+      port:
+        targetPort: sonarqube
+      tls:
+        termination: edge
+      to:
+        kind: Service
+        name: sonarqube
+  - apiVersion: v1
+    data:
+      username: YWRtaW4=
```
Comment (Fosol) [inline openshift/4.0/templates/sonarqube/sonarqube-postgresql-template.yaml:45]: don't change, but probably should be configurable parameter

File: maintenance/RESTORE_BACKUPS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# How to Restore a backup hosted on an external openshift instance
+
+**NOTE: if you are simply trying to restore a backup from the same openshift instance, skip directly to step 4**
+**NOTE: if performing a restore on a live instance of PIMS, see maintenance page documentation [here](./README.md)**
+
+0. Optionally, enable the display of the maintenance page if the restore is targeting a live instance of PIMS.
+1. login to the openshift environment hosting the DB you wish to copy the backup from.
+1. change to the project hosting the desired database using: oc project \${projectName}
```
Comment (Fosol) [inline maintenance/RESTORE_BACKUPS.md:8]: Would be nice to use code Markdown for any CLI commands.
`oc project \${projectName}`

**Review by Fosol — APPROVED** [review]

File: maintenance/RESTORE_BACKUPS.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+1. use oc-rsync to copy the desired backup files from the database to a directory on your local machine. ie: oc rsync pims-db-prod-6-ddck4:/backups/daily/2021-02-18 ./backups
+1. login to the openshift environment hosting the DB where you would like to restore the backup.
+1. change to the project hosting the desired database using: oc project \${projectName}
+1. use oc-rsync to copy the backup files from your local machine to the volume on the backup pod. ie: oc rsync ./backups pims-database-1-k8hsh:/backups/daily/2021-02-18
+1. login to the terminal on the backup pod and backup the database using ./backup.sh. ie: ./backup.sh -r mssql:pims-database/pims -f pims-db-prod-pims_2021-02-18_01-00-00 **NOTE: you may exclude the -f parameter if restoring a backup from the same openshift instance**
```
Comment (devinleighsmith) [inline maintenance/RESTORE_BACKUPS.md:13]: this should be mssql=

**Comment by devinleighsmith** [comment]
all pipelines now passing: https://jenkins-354028-tools.apps.silver.devops.gov.bc.ca/job/354028-tools/

File: maintenance/RESTORE_BACKUPS.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+**NOTE: if you are simply trying to restore a backup from the same openshift instance, skip directly to step 4**
+**NOTE: if performing a restore on a live instance of PIMS, see maintenance page documentation [here](./README.md)**
+
+0. Optionally, enable the display of the maintenance page if the restore is targeting a live instance of PIMS.
+1. login to the openshift environment hosting the DB you wish to copy the backup from.
+1. change to the project hosting the desired database using: oc project `\${projectName}`
+1. use oc-rsync to copy the desired backup files from the database to a directory on your local machine. ie: `oc rsync pims-db-prod-6-ddck4:/backups/daily/2021-02-18 ./backups`
+1. login to the openshift environment hosting the DB where you would like to restore the backup.
+1. change to the project hosting the desired database using: oc project `\${projectName}`
+1. use oc-rsync to copy the backup files from your local machine to the volume on the backup pod. ie: `oc rsync ./backups pims-database-1-k8hsh:/backups/daily/2021-02-18`
```
Comment (Fosol) [inline maintenance/RESTORE_BACKUPS.md:12]: I need to read up on `oc rsync`

**Review by jdtoombs — APPROVED** [review]

**Pull request merged**